build.gradle.kts

```diff
@@ -0,0 +1,131 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import com.github.jengelman.gradle.plugins.shadow.transformers.ServiceFileTransformer
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "no.nav.integrasjon"
+version = "0.74"
+
+val kotlinVersion = "1.3.21"
+val kotlinLogginVersion = "1.6.25"
+val ktorVersion = "1.1.3"
+
+// Could not find usage for this? but leave it in
+// val coroutinesVersion = "0.22.5"
+
+val jacksonDatatypeVersion = "2.9.8"
+
+val kafkaVersion = "2.0.0"
+val embeddedkafkaVersion = "2.1.1"
+
+val prometheusVersion = "0.6.0"
+val logstashEncoderVersion = "5.3"
+val logbackVersion = "1.2.3"
+val log4jVersion = "1.7.25"
+
+val unboundidVersion = "4.0.10"
+
+val swaggerVersion = "3.22.0"
+
+val spekVersion = "2.0.2"
+val kluentVersion = "1.49"
+
+val konfigVersion = "1.6.10.0"
+
+val mainClass = "no.nav.integrasjon.MainKt"
```

Comment:
Maybe move to mainClass tasks, like this:
  withType<Jar> {
        manifest.attributes["Main-Class"] = "no.nav.integrasjon.MainKt"
    }